docs/recording/use_sim_time.md

```diff
@@ -0,0 +1,52 @@
+# sim_time での時系列データ評価
```

Comment:
@takeshi-iwanari  
Have you located this kind of page in your merge?

Reply:
Yes, the recording section has the page to explain sim_time.
https://tier4.github.io/CARET_doc/pr-77/recording/sim_time/

Reply:
I deleted "use_sim_time.md".
"sim_time.md" is stayed.